JS.md

```diff
@@ -0,0 +1,35 @@
+# JS client-side conventions
+
+JavaScript is often difficult to get started with because the landscape (and
+language) is vast, and the community has less adherence to conventions.
+
+This is a conventions document for not getting into trouble in JS.
+
+In the future, we will have a little js pack repo for copying in a good set
+of standards into a new application.
+
+## React
+
+* Use `jsx` to build templates
+* Create templates that are as declarative as possible
```

Comment:
This is probably a "Jacob is a React newbie" thing, but I don't really understand this. Is this something that someone more well-versed in React would understand? Or would it benefit from an example?

Reply:
It seems like the rest of the repo is just a bullet list, so maybe I don't need to build out these conventions. It was hard to find a dividing line between describing in depth the kinds of things that go wrong with React and not enough information on the conventions.

I could definitely put in some examples too, but maybe this whole file could go or get slimmer.

Thoughts?

Reply:
Yeah, dunno. Maybe just keep this as-is, but make it a topic for eng guild some day?

Reply:
✅